tanner/tests/test_dorks_manager.py

```diff
@@ -0,0 +1,90 @@
+import unittest
+import asyncio
+import aioredis
+import random
+from unittest import mock
+from tanner import redis_client
+from tanner import config
+from tanner.utils.asyncmock import AsyncMock
+from tanner.dorks_manager import DorksManager
+
+
+class TestDorksManager(unittest.TestCase):
+    def setUp(self):
+        self.loop = asyncio.new_event_loop()
+        self.redis_client = None
+        self.path = None
+
+        async def connect():
+            self.redis_client = await redis_client.RedisClient.get_redis_client()
+
+        self.loop.run_until_complete(connect())
+
+        self.handler = DorksManager()
+        self.returned_result = None
+        self.expected_result = None
+
+    def test_push_init_dorks(self):
+
+        async def test():
+            await self.handler.push_init_dorks(config.TannerConfig.get('DATA', 'dorks'), self.handler.dorks_key,
+                                               self.redis_client)
+            self.returned_result = await self.redis_client.smembers(self.handler.dorks_key)
+
+        self.loop.run_until_complete(test())
+        self.assertEqual(len(self.returned_result), 3562)
```

Comment:
Is `self.handler.dorks_key` == `3562`?
Then I guess you should substitute `self.handler.dorks_key` with some constant and check it here other wise tests will fail if there is some modification in `self.handler.dorks_key`

Reply:
No, `self.handler.dorks_key` is not equal to 3562. 3562 is the total length of the list which is returned by `smembers`. Here `self.handler.dorks_key` == `9445102ddf663ed0a20382c219a952ba`

Reply:
I guess total length should remain same.

Reply:
As far I understand this is loaded from the pickle which can change.